kerosene/datasets/dataset.py

```diff
@@ -40,7 +40,7 @@ def fuel_data_to_list(fuel_data, shuffle):
     else:
         scheme = SequentialScheme(fuel_data.num_examples, fuel_data.num_examples)
     fuel_data_stream = DataStream.default_stream(fuel_data, iteration_scheme=scheme)
-    return fuel_data_stream.get_epoch_iterator().next()
+    return next(fuel_data_stream.get_epoch_iterator())
```

Comment:
Yes, I believe it is. 

The get_epoch_iterator method [returns a DataIterator object](https://github.com/mila-udem/fuel/blob/master/fuel/streams.py#L98), which extends the [six.Iterator class](https://pythonhosted.org/six/index.html#six.Iterator). 

The [six.Iterator object](https://pythonhosted.org/six/index.html#six.Iterator) is intended to implement an `obj.next()` method that will always call the built-in `obj.__next__()` method, so as to implement a Python 3-pythonic iterator without breaking Python 2.

Via link: " In Python 2, Iterator has one method: next. It simply delegates to __next__."

The DataIterator class [implements a `__next__` method](https://github.com/mila-udem/fuel/blob/master/fuel/iterator.py#L28) that is called by either Python 2 or Python 3.

Here's a [quick Gist demo](https://gist.github.com/charlesreid1/5521aadc3cdb492f21b8799693a10c1b).

This will affect dribnet/kerosne/pull#10 but only to shorten it from four lines to one, and get rid of that ugly version check. :)